package.json

```diff
@@ -20,7 +20,7 @@
         "update-codeowners": "node scripts/update-codeowners.js",
         "test-all": "node node_modules/@definitelytyped/dtslint-runner/dist/index.js --path .",
         "test": "dtslint types",
-        "lint": "dtslint types",
+        "lint": "dtslint types/scrivito",
```

Comment:
please revert this change

`dtslint` has a quirky/undiscoverable interface: it accepts `types/scrivito` or `types scrivito`. The latter lets you write `npm test scrivito`

Reply:
Ah, I didn't see how this was supposed to work. Sorry for pushing my changes